### ISSUE TITLE
[Bug?]: No notification for group members

### ISSUE THREAD

**Issue by nreojb** [fguillot/kanboard#1616]
I'm not sure if this is a bug or intended but I think it could a bug.

When I add a group of users to a project no one gets a notification when a notification is usually triggered.
(Notification is of course activated with all users of this group!).

For example:
* There is a group called "Sales"
* The users Mike and Molly are part of this group
* This group is now a "Allowed Group" of the project "BigWin"
* Ruben is the project manager of this project
* Ruben adds a new task and change the "Assignee" to "Mike"

And there it is - Mike is getting no notification.
But when I add Mike to the "Allowed Users" section he is getting a notification!

So in conclusion:
* When a user is directly added to the section "Allowed Users" he is going to get a notification.
* BUT if a user is only added to a project with a GROUP then he is getting no notification.

Is this intended?

Thank you for your great work and happy new year :-)

John

**Comment by fguillot** [comment]
This feature has been added to the development branch. Thanks.
